Dockerfile

```diff
@@ -18,9 +17,23 @@ RUN make build RELEASE_TAG=${release_tag}
 # ubi8:latest
 FROM registry.access.redhat.com/ubi8/ubi:latest
 ARG quay_expiration
+ARG release_tag
+ARG preflight_commit
 ARG ARCH
 ARG OS
 
+# Metadata
+LABEL name="Preflight" \
+      vendor="Red Hat, Inc." \
+      maintainer="Red Hat OpenShift Ecosystem" \
+      version="1" \
```

Comment:
Non-Blocking: What's the difference between `version` and `release_tag` in context of `metadata`? ie when should version here change?

Reply:
If I'm seeing this correctly, I would think when the Dockerfile changes. But @komish would be better to answer this.

Reply:
That makes sense to me.